lib/chroma-docker-image-ecr-kms-deployment-cdk-stack.ts

```diff
@@ -27,11 +28,14 @@ export class ChromaDockerImageEcrDeploymentCdkStack extends cdk.Stack {
         ecrRepository.addLifecycleRule({ maxImageAge: cdk.Duration.days(7), rulePriority: 1, tagStatus: ecr.TagStatus.UNTAGGED }); // delete images older than 7 days
         ecrRepository.addLifecycleRule({ maxImageCount: 4, rulePriority: 2, tagStatus: ecr.TagStatus.ANY }); // keep last 4 images
 
-        // Copy from docker registry to ECR.
-        new ecrDeploy.ECRDeployment(this, `${props.appName}-${props.environment}-ECRDeployment`, {
-            src: new ecrDeploy.DockerImageName('chromadb/chroma:latest'),
-            dest: new ecrDeploy.DockerImageName(`${ecrRepository.repositoryUri}:${props.imageVersion}`),
-        });
+        const deployImageVersions = props.imageVersion === LATEST_IMAGE_VERSION? [props.imageVersion] : [props.imageVersion, LATEST_IMAGE_VERSION];
+        for (const deployImageVersion of deployImageVersions) {
+            // Copy from docker registry to ECR.
+            new ecrDeploy.ECRDeployment(this, `${props.appName}-${props.environment}-${deployImageVersion}-ECRDeployment`, {
+                src: new ecrDeploy.DockerImageName('chromadb/chroma:latest'),
```

Comment:
**Suggestion:** To avoid hardcoding the Docker image name, consider passing it as a parameter in `props` and using it in the `ECRDeployment` configuration.
> 🎯  **High Priority** 
 > [category: **`best practice`**, importance: **`8`**]
> **Reason:** Using a parameter for the Docker image name instead of hardcoding enhances flexibility and maintainability of the code.
 

```suggestion
                src: new ecrDeploy.DockerImageName(props.dockerImageName),
```